providers/google/src/airflow/providers/google/cloud/sensors/gcs.py

```diff
@@ -63,6 +63,8 @@ class GCSObjectExistenceSensor(BaseSensorOperator):
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
     :param retry: (Optional) How to retry the RPC
+    :param user_project: The identifier of the Google Cloud project to bill for the request.
+        Required for Requester Pays buckets.
```

Comment:
This feels like something that needs to be in some kind of a base class. This is relevant for several GCS sensors isn't it?

Reply:
Correct, I think it should be `project_id` parameter, if I right understand the reason of this PR